.github/workflows/test-indexer-debian.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Wazuh Indexer upon upgrade - Debian
+on:
+  pull_request:
+    paths:
+      - 'stack/indexer/deb/debian/postrm'
+      - 'stack/indexer/deb/debian/postinst'
+      - 'stack/indexer/deb/debian/preinst'
+  workflow_dispatch:
```

Comment:
The intention of the workflow is to be executed when the SPEC files are changed in a PR. With this, I think the parameter should be kept. A better solution for this is to consider every SPEC file of the `debian` folder, and not only `postrm`, `postinst` and `preinst`. So the new path is `stack/indexer/deb/debian/*`

Reply:
In the `workflow_dispatch` event definition, you used `:`, which means you will enumerate parameters. 
I don't know if it works like this:
```
  pull_request:
    paths:
      - 'stack/indexer/deb/debian/*'
  workflow_dispatch
  ```
Instead of 
```
  pull_request:
    paths:
      - 'stack/indexer/deb/debian/*'
  workflow_dispatch:
```

Reply:
If the only event that triggers the workflow is `workflow_dispatch` it can be specified as:
`on: [workflow_dispatch]`

In this case, there are more events apart from `workflow_dispatch`, so it's not possible to delete the `:`.